Microsoft.Toolkit.Uwp.UI.Animations/Helpers/TransitionHelper.Properties.cs

```diff
@@ -0,0 +1,120 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// A animation helper that morphs between two controls.
+    /// </summary>
+    public sealed partial class TransitionHelper
+    {
+        private FrameworkElement _source;
+        private FrameworkElement _target;
+        private List<AnimationConfig> _animationConfigs = new();
+
+        /// <summary>
+        /// Gets or sets the source control.
+        /// </summary>
+        public FrameworkElement Source
+        {
+            get
+            {
+                return this._source;
+            }
+
+            set
+            {
+                this._source = value;
+                this.IsTargetState = false;
+                this.UpdateSourceAnimatedElements();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the target control.
+        /// </summary>
+        public FrameworkElement Target
+        {
+            get
+            {
+                return this._target;
+            }
+
+            set
+            {
+                this._target = value;
+                this._needUpdateTargetLayout = true;
+                this.UpdateTargetAnimatedElements();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the collection of animation configurations of UI elements that need to be connected by animation.
+        /// </summary>
+        public List<AnimationConfig> AnimationConfigs
```

Comment:
If none of these child elements will contain bindings, you should be able to set a ContentPropertyAttribute on the class to specify this property, then your XAML won't need to call out the property and can be just this:
```xml
<ani:TransitionHelper x:Key="MyTransitionHelper">
    <ani:AnimationConfig Id="background" ScaleMode="Scale" />
    <ani:AnimationConfig Id="image" ScaleMode="Scale" />
    <ani:AnimationConfig Id="guide" />
    <ani:AnimationConfig Id="name" ScaleMode="ScaleY" />
    <ani:AnimationConfig Id="desc" />
</ani:TransitionHelper>
```

Otherwise if the child elements would need bindings, we'd probably have to inherit from DependencyObjectCollection like we needed for `AnimationSet` and `AnimationScope`.

Eh, @Sergio0694?

Reply:
I added `ContentProperty` attribute for `TransitionHelper`, I think it should not need bindings, `ContentProperty` is ok for it.